crates/proc-macro-srv/src/lib.rs

```diff
@@ -59,9 +63,24 @@ impl ProcMacroSrv {
 
         let macro_body = task.macro_body.to_subtree();
         let attributes = task.attributes.map(|it| it.to_subtree());
-        let result = expander
-            .expand(&task.macro_name, &macro_body, attributes.as_ref())
-            .map(|it| FlatTree::new(&it));
+        let result = crossbeam::scope(|s| {
```

Comment:
Comment to replace this (and the crossbeam dependency altogether therefor) with the std version once stable would be nice